app/models/cms/email_template.rb

```diff
@@ -50,7 +50,25 @@ def self.reset_templates_path!
   EMAIL_FORMAT = /(?:(#{EMAIL_ADDRESS_FORMAT})|#{EMAIL_WITH_NAME_FORMAT})\s*/i
   EMAILS_FORMAT = /\A#{EMAIL_FORMAT}(,\s*#{EMAIL_FORMAT})*\Z/i
 
-  serialize :options, Hash
+  class HashOrParameters < ::ActiveRecord::Coders::YAMLColumn
+    def load(string)
+      obj = super(string)
+      obj.respond_to?(:to_unsafe_h) ? obj.to_unsafe_h : obj.to_h if obj
+    end
+
+    def dump(object)
+      return object if object.is_a?(String) && assert_valid_value(load(object))
```

Comment:
In fact this line does a lot more than needed
Now `options=` can take a `String` as argument. Before, it rejected anything.

I made this possible because of this test https://github.com/3scale/porta/pull/1679/files#diff-00583319eebb023a454123e16fbd9655R51

Without it I cannot use `update_column :options, '--- some yaml'`, because Rails always tries to serialize the attribute ...
And there is no way I can do it simply, without using low level Rails API ...